horreum-backend/src/main/java/io/hyperfoil/tools/horreum/server/SecurityBootstrap.java

```diff
@@ -95,4 +111,33 @@ private void addTeamMembership(UserInfo userInfo, String teamName, TeamRole role
         Optional<Team> storedTeam = Team.find("teamName", teamName).firstResultOptional();
         userInfo.teams.add(new TeamMembership(userInfo, storedTeam.orElseGet(() -> Team.getEntityManager().merge(new Team(teamName))), role));
     }
+
+    // --- //
+
+    /**
+     * Create an admin account if there are no accounts in the system.
+     * The account should be removed once other accounts are created.
+     */
+    public void checkBootstrapAccount() {
+        // checks the list of administrators. a user cannot remove himself nor create the bootstrap account (restricted namespace)
+        List<String> administrators = backend.get().administrators().stream().map(userData -> userData.username).toList();
+        if (administrators.isEmpty()) {
+            UserService.NewUser user = new UserService.NewUser();
+            user.user = new UserService.UserData("", BOOTSTRAP_ACCOUNT, "Bootstrap", "Acount", "horreum@example.com");
+            user.password = providedBootstrapPassword.orElseGet(() -> getLaunchMode().isDevOrTest() ? "secret" : generateRandomPassword(RANDOM_PASSWORD_DEFAULT_LENGTH));
+
+            backend.get().createUser(user);
+            Log.infov("Created temporary account {0} with password {1}", BOOTSTRAP_ACCOUNT, user.password);
```

Comment:
I think this needs to be more prominent in the log
```suggestion
            Log.infov("\n\n*********************************\nCreated temporary account `{0}` with password `{1}`\n*********************************", BOOTSTRAP_ACCOUNT, user.password);
```

creates the following output:

```
2024-06-13 16:05:41,982 INFO  [io.sma.rea.mes.amqp] (vert.x-eventloop-thread-1) SRMSG16203: AMQP Receiver listening address dataset-event
2024-06-13 16:05:42,306 INFO  [io.hyp.too.hor.ser.SecurityBootstrap] (Quarkus Main Thread) 

*********************************
Created temporary account `horreum.bootstrap` with password `secret`
*********************************

2024-06-13 16:05:42,389 INFO  [io.hyp.too.hor.svc.use.KeycloakUserBackend] (Quarkus Main Thread) Added administrator role to user horreum.bootstrap

```

Reply:
makes sense. I changed to something similar (3 lines in the output)